echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/PubsubSubscriptionController.java

```diff
@@ -16,6 +16,8 @@
 
 package com.netflix.spinnaker.echo.pubsub;
 
+import java.util.List;
```

Comment:
I think normally the `java.util.*` goes at the end in our repos

Reply:
Boo intellij formatter...